tests/agent/test_docker_agent.py

```diff
@@ -222,6 +222,37 @@ def test_populate_env_vars_sets_log_to_cloud(flag, api, config_with_api_key):
     assert env_vars["PREFECT__LOGGING__LOG_TO_CLOUD"] == str(not flag).lower()
 
 
+@pytest.mark.parametrize(
```

Comment:
@madkinsz Before I implement for the rest of the agents, is this what you had in mind?

Reply:
I'd rename them for clarity to: `disabled_on_agent,enabled_by_run,expected`

Otherwise yeah this lgtm,